europe/README.EUROPE

```diff
@@ -294,3 +294,111 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
 Recommended to transform from Portuguese Datum 73 (EPSG:27493) to ETRS89 in Portugal (EPSG:3763).
 
 * D73_ETRS89_geo.gsb
+
+### Iceland: ISN93 -> ISN2016
+
+2D geodetic transformation from Iceland Datum ISN93 to Iceland Datum ISN2016
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+The transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+
+Recommended to transform from ISN93 2D geodetic (EPSG:4659) to ISN2016 2D geodetic (EPSG:8086), also this transformation has code in EPSG Dataset (EPSG:9232)
```

Comment:
great. Could you open a pull request against the OSGeo/PROJ repository to register this grid in the data/sql/grid_alternatives.sql file ? (mostly a matter to register the fact that the grid will be in the proj-datumgrid-europe package )

Reply:
Opened the new pull request for grid_alternatives.sql